packages/crawler/README.md

```diff
@@ -0,0 +1,43 @@
+# Porsche Design System Crawler
+
+Porsche Design System Crawler is designed to crawl over all relevant portfolio frontpages and make report about:
+
+- Used PDS version(s) and PDS prefixes
+- Used PDS components and configuration per component
+- PDS components and properties which are never used
+- Aggregated data - amout of PDS components and properties
+- Information about PDS components' slot's contents (children)
+- Information about parent PDS component
+
+## Start Crawler
+
+```
+yarn start:crawler
```

Comment:
Isn't it just `yarn start` from this location?

Reply:
true, fixed